src/plugins/buflist/buflist-config.c

```diff
@@ -773,6 +776,30 @@ buflist_config_init ()
         NULL, NULL, NULL,
         &buflist_config_change_buflist, NULL, NULL,
         NULL, NULL, NULL);
+    buflist_config_format_tls_version = weechat_config_new_option (
+        buflist_config_file, ptr_section,
+        "tls_version", "string",
+        N_("format for tls_version on an IRC server buffer "
+           "(note: content is evaluated, see /help buflist)"),
+        NULL, 0, 0,
+        " ${color:default}(${if:${tls_version}==TLS1.3?${color:green}:"
+        "${if:${tls_version}==TLS1.2?${color:yellow}:${color:red}}}"
+        "${tls_version}${color:default})",
+        NULL, 0,
+        NULL, NULL, NULL,
+        &buflist_config_change_buflist, NULL, NULL,
+        NULL, NULL, NULL);
+    buflist_config_format_cleartext = weechat_config_new_option (
+        buflist_config_file, ptr_section,
+        "format_cleartext", "string",
+        N_("format for indicating an unencrypted connection on an IRC "
+           "server buffer (note: content is evaluated, see /help buflist)"),
+        NULL, 0, 0,
+        " ${color:default}(${color:red}cleartext${color:default})",
```

Comment:
This string must be translated (for the word `cleartext`).
But that's not ideal to have the whole format translated, as the default value would change according to your locale.
I'll think about a better way to translate this.

Reply:
I've created a buflist_config_format_tls_format_default buffer that I print a gettext("cleartext") into. This does make the default translation dependent which is kind of weird, but if you think up a better approach I am happy to change it.